### DISCUSSION TITLE
`Unable to create process` error while running `pipenv install atri`

### DISCUSSION THREAD

**Discussion by RahulJava-1** [Atri-Labs/atrilabs-engine#411] [Help: Installation & Start] [answered]
Installing atri...
Error:  An error occurred while installing atri!
Error text:
Unable to create process using C:\Users\XXXXXX\AppData\Local\Programs\Python\Python310\python.exe "C:\Users\XXXXXX\AppData\Roaming\Python\Python310\site-packages\pipenv\patched\pip\__pip-runner__.py" install --no-input --upgrade --exists-action=i -r c:\temp\pipenv-8yerm4u5-requirements\pipenv-a_lzzjk1-requirement.txt -i https://pypi.org/simple'

Installation Failed

**Comment by darshitac11** [comment]
@RahulJava-1 - thanks for raising this issue.

Please check the following:

1. [All prerequisites are installed](https://docs.atrilabs.com/getting-started/installation/install_prereqs).
2. If you are using `pipenv`, please ensure that it is added to the PATH environment variable.

**Reply by RahulJava-1** [reply]
I have already installed all prerequisites but when I type pipenv install atri and hit enter then this error is raised.

**Reply by darshitac11** [reply]
Have you checked if it is in the `PATH`?

**Reply by RahulJava-1** [reply]
> Have you checked if it is in the `PATH`?

Yes ma'am I checked in path variables.

**Reply by darshitac11** [reply]
Just for clarification, this is the complete message you received in the terminal, right?

**Reply by darshitac11** [reply]
@cruxcode - can you help here?

**Reply by RahulJava-1** [reply]
> Just for clarification, this is the complete message you received in the terminal, right?

Yes

**Reply by cruxcode** [reply]
@RahulJava-1 please try to install it again. It doesn't seem to be a problem on our end. If it doesn't get installed again, you might have to upgrade your python/pipenv.

**Comment by darshitac11** [comment, marked as answer]
Run `pipenv install atri` again. If it still doesn't work, upgrade your python/pipenv.

**Comment by Shubhodeep100** [comment]
`python -m pipenv install atri`
try running the above command. `pipenv install atri` was't working for me as well.
